src/main/java/mate/hwdao/dao/impl/jdbc/ManufactureDaoJdbcImpl.java

```diff
@@ -0,0 +1,115 @@
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Saving manufacturer " + manufacturer
+                    + " failed", ex);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        Manufacturer manufacturer = null;
+        String query = "SELECT * FROM manufacture WHERE id = ? AND is_exist = true";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection().prepareStatement(query)) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturer = getManufacturer(resultSet);
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get manufacture with " + id, ex);
+        }
+        return Optional.ofNullable(manufacturer);
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        Manufacturer manufacturer = null;
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String query = "SELECT * FROM manufacture WHERE is_exist = true";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection().prepareStatement(query)) {
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException("Can't get list of manufacturers because of ", ex);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacture SET name = ?, country = ?"
+                + " WHERE id = ? AND is_exist = true";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection().prepareStatement(query)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            if (preparedStatement.executeUpdate() > 0) {
+                return manufacturer;
+            }
+        } catch (SQLException ex) {
+            throw new DataProcessingException(manufacturer.toString()
+                    + "was not updated, because of ", ex);
+        }
+        throw new RuntimeException("Can't find manufacturer with id " + manufacturer.getId());
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE manufacture SET is_exist = false WHERE id = ?";
```

Comment:
```suggestion
        String query = "UPDATE manufacture SET exists = false WHERE id = ?";
```
in eng you can't say "is exist", the correct version is "does something exist? - yes it exists"